util/password/password.go

```diff
@@ -47,6 +53,13 @@ func verifyPasswordWithHashers(password, hashedPassword string, hashers []Passwo
 			break
 		}
 	}
+
+	// Even though good hashers will disallow blank passwords, let's be explicit that ALL BLANK PASSWORDS ARE INVALID.  Full stop.
+	// This goes AFTER verification to mitigate potential for timing attacks.
```

Comment:
I think it makes sense execute empty password check before checking all hashers

Reply:
👍